File: SEMANA 1/TALLER 1/Ejercicio3.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
#include <string>
using namespace std;

int main() {
    srand(time(0));

    const int N = 25;
    string sucursales[N];
    double ventas[N];
    double suma = 0.0;

    for (int i = 0; i < N; i++) {
        sucursales[i] = "Sucursal " + to_string(i + 1);
        ventas[i] = 1000 + rand() % 9001;
        suma += ventas[i];
    }

    double promedio = suma / N;

    cout << "Promedio de ventas: " << promedio << endl;
    cout << "Sucursales por encima del promedio:" << endl;
    for (int i = 0; i < N; i++) {
        if (ventas[i] > promedio) {
            cout << sucursales[i] << ": " << ventas[i] << endl;
        }
    }

    return 0;
}

File: SEMANA 1/TALLER 1/Ejercicio3.py
import random

N = 25
sucursales = [f"Sucursal {i + 1}" for i in range(N)]
ventas = [random.randint(1000, 10000) for _ in range(N)]

promedio = sum(ventas) / N

print(f"Promedio de ventas: {promedio:.2f}")
print("Sucursales por encima del promedio:")
for i in range(N):
    if ventas[i] > promedio:
        print(f"{sucursales[i]}: {ventas[i]}")